README.md

```diff
@@ -6,28 +6,13 @@
 
 ESLinting for Ember CLI apps, [ESLint](http://eslint.org/) provides a scriptable mechanism for checking applications against a strict set of rules.
 
-## Installation
-```
-ember install ember-cli-eslint
-```
-This will create a `.eslintrc` file in your project directory setup with the correct parser.
-
-## ESLint Babel
-
-The [`babel-eslint`](https://github.com/babel/babel-eslint) parser will solve any new ECMA missing functionality in ESLint.
-
-Due to an [issue that occurred as the result of changes across `eslint` and `babel-eslint`](https://github.com/babel/babel-eslint/issues/267), versions of `babel-eslint` less than `6.0.0-beta.6` will be incompatible with versions of `eslint` greater than or at `2.4.0` -- which this project relies on after `1.2.x`.
-
-As such, it's recommended that you install `babel-eslint` at a version >= `6.0.0-beta.6`:
+## Basic setup
 
 ```
-npm install --save-dev babel-eslint@6.0.0-beta.6
+ember install ember-cli-eslint
 ```
 
-After installing, simply add the following option to your `eslint` configuration file:
-```
-    "parser": "babel-eslint",
-```
+This will create a `.eslintrc.js` file in the root of your project, and another `.eslintrc.js` file inside of `/test`. These files extend from our recommended configurations for [Ember application code](/best-practices/ember-application) and [Ember testing code](/best-practices/ember-test), respectively. However, starting from scratch is as easy as deleting the `extends` declaration and [writing your own configuration rules as usual](http://eslint.org/docs/user-guide/configuring).
```

Comment:
Would it make more sense to make those full URLs instead?  That way they continue to work if someone is readying the `README` locally or from the [readme page](https://github.com/ember-cli/ember-cli-eslint/blob/master/README.md).